test/bats/plugin-test.bats

```diff
@@ -159,11 +159,8 @@ SLEEP_TIME=1
     run kubectl apply -f ./config/samples/config_v1beta1_verifier_sbom.yaml
     sleep 5
     run kubectl apply -f ./config/samples/config_v1beta1_verifier_complete_licensechecker.yaml
-
-    # Skipping test for now until expected usage/configuration of this plugin can be verified
-    # sleep 5
-    # run kubectl apply -f ./config/samples/config_v1beta1_verifier_schemavalidator.yaml
-    # sleep 5
+    run kubectl apply -f ./config/samples/config_v1beta1_verifier_schemavalidator.yaml
```

Comment:
why have we reenabled this one?

Reply:
Actually I'm also wondering why this was disabled though it can pass the verification. As for the reason, the new result will include reports for all artifacts even though we don't have a verifier configured for some artifact. In this test case, there will be an empty `verifierReports` for the artifactType: `vnd.aquasecurity.trivy.report.sarif.v1`, which failed the rego evaluation. However, the old result will only include reports generated by configured verifiers, so it will just skip the corresponding verification and return success.